extensions/amp-story/1.0/amp-story.js

```diff
@@ -1064,10 +1067,51 @@ export class AmpStory extends AMP.BaseElement {
       });
     }
 
+    this.buildScreenReaderBackButton_();
+
     // Will resolve when all pages are built.
     return storyLayoutPromise;
   }
 
+  /**
+   * Creates a focusable button for screen readers to navigate back.
+   * @private
+   */
+  buildScreenReaderBackButton_() {
+    const backButton = document.createElement('button');
+
+    const label = this.localizationService_.getLocalizedString(
+      LocalizedStringId.AMP_STORY_PAGINATION_BUTTON_PREVIOUS_PAGE_LABEL
+    );
+
+    backButton.classList.add('i-amphtml-story-screen-reader-back-button');
+    label && backButton.setAttribute('aria-label', label);
+    this.mutateElement(() => {
+      this.element.appendChild(backButton);
+    });
+
+    backButton.addEventListener('click', () => {
+      this.storeService_.dispatch(
+        Action.SET_ADVANCEMENT_MODE,
+        AdvancementMode.MANUAL_ADVANCE
+      );
+      this.previous_();
+    });
+
+    // Append class to hide button if on first page.
+    this.storeService_.subscribe(
+      StateProperty.CURRENT_PAGE_INDEX,
+      (pageIndex) =>
+        this.mutateElement(() =>
+          backButton.classList.toggle(
+            'i-amphtml-story-screen-reader-back-button-hidden',
+            pageIndex === 0
```

Comment:
Can you also add `!this.viewer_.hasCapability('swipe')` ? We might need this button to show if we are in a player/viewer environment. Tapping back on the first page should go to the previous story.